harold/plugins/deploy.py

```diff
@@ -237,7 +237,7 @@ class DeploySendAnnouncementListener(DeployListener):
 
     def _handle_request(self, request):
         message = request.args['message'][0]
-        self.monitor.announce(self.monitor.irc, 'harold', None, message)
+        self.monitor.announce(self.monitor.irc.bot, 'harold', None, message)
```

Comment:
Related commit:
https://github.com/spladug/harold/commit/c7dfdde2f9d503f9d38ca10b29bfa7f1e5dd0840

Fixes this exception:
```
2019-10-08 02:16:48+0000 [-] Unhandled error in Deferred:
2019-10-08 02:16:48+0000 [-] Unhandled Error
	Traceback (most recent call last):
	  File "/home/ubuntu/venv/harold/local/lib/python2.7/site-packages/twisted/internet/defer.py", line 393, in callback
	    self._startRunCallbacks(result)
	  File "/home/ubuntu/venv/harold/local/lib/python2.7/site-packages/twisted/internet/defer.py", line 501, in _startRunCallbacks
	    self._runCallbacks()
	  File "/home/ubuntu/venv/harold/local/lib/python2.7/site-packages/twisted/internet/defer.py", line 588, in _runCallbacks
	    current.result = callback(current.result, *args, **kw)
	  File "/home/ubuntu/venv/harold/local/lib/python2.7/site-packages/twisted/internet/defer.py", line 1184, in gotResult
	    _inlineCallbacks(r, g, deferred)
	--- <exception caught here> ---
	  File "/home/ubuntu/venv/harold/local/lib/python2.7/site-packages/twisted/internet/defer.py", line 1128, in _inlineCallbacks
	    result = g.send(result)
	  File "/home/ubuntu/src/harold/harold/plugins/deploy.py", line 1032, in announce
	    irc.send_message(salon.channel, ":siren: ANNOUNCEMENT FROM @%s: %s" % (
	exceptions.AttributeError: 'SlackPlugin' object has no attribute 'send_message'

```